ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-master.yaml

```diff
@@ -78,6 +80,9 @@ tests:
         export KUBE_SSH_KEY_PATH=${cluster_profile}/ssh-privatekey
 
         make run-wmcb-ci-e2e-test
+      dependencies:
+      - env: WMCB_IMAGE
+        name: windows-machine-config-bootstrapper-tests
```

Comment:
Please clarify why you need an override given you are overriding with the same value.

Reply:
actually we do not need to override. Confirmed and removed it.